src/wallet/wallet.cpp

```diff
@@ -3840,40 +3847,129 @@ bool CWallet::CreateTransaction(const std::vector<CRecipient>& vecSend, CWalletT
                         }
                         return false;
                     }
+                    vecCoins.assign(setCoinsTmp.begin(), setCoinsTmp.end());
+                }
+
+                // Fill vin
+                //
+                // Note how the sequence number is set to max()-1 so that the
+                // nLockTime set above actually works.
+                txNew.vin.clear();
+                for (const auto& coin : vecCoins) {
+                    CTxIn txin = CTxIn(coin.outpoint,CScript(),
+                                              CTxIn::SEQUENCE_FINAL - 1);
+                    txNew.vin.push_back(txin);
+                }
+
+                auto calculateFee = [&](CAmount& nFee) -> bool {
+                    // Fill in dummy signatures for fee calculation.
+                    int nIn = 0;
+                    for (const auto& coin : vecCoins)
+                    {
+                        const CScript& scriptPubKey = coin.txout.scriptPubKey;
+                        SignatureData sigdata;
+                        if (!ProduceSignature(DummySignatureCreator(this), scriptPubKey, sigdata))
+                        {
+                            strFailReason = _("Signing transaction failed");
+                            return false;
+                        } else {
+                            UpdateTransaction(txNew, nIn, sigdata);
+                        }
```

Comment:
fix formatting please (brackets same line)

Reply:
done in fdf3829b629182a4c2ab0a9df59b98954d885110